app/controllers/charges_controller.rb

```diff
@@ -34,6 +34,10 @@ def create
 
   def update
```

Comment:
you should probably use the destroy action instead, since you're technically destroying a subscription.